pydra/engine/task.py

```diff
@@ -209,7 +209,8 @@ def command_args(self):
             pos_args.append(pos_args.pop(0))
         # dropping the position index
         cmd_args = []
-        [cmd_args.extend(el[1]) for el in pos_args]
+        for el in pos_args:
+            cmd_args += el[1]
```

Comment:
i think `+=` appends, so if `el[1]` is a list, it will append a list, but the previous command would have appended all the elements of the list. so just making sure that's your intent.

Reply:
`+=` is not the same as `append`, `ll=[1]; ll+=[2,3]; all==[1,2,3]`